pycheribuild/projects/cross/cheribsd.py

```diff
@@ -1194,16 +1194,22 @@ def __init__(self, config: CheriConfig):
         self.extra_kernels_with_mfs = []
         if self.buildFpgaKernels:
             if self.compiling_for_mips(include_purecap=True):
-                if self.crosscompile_target.is_hybrid_or_purecap_cheri():
-                    prefix = "CHERI_DE4_"
+                if self.purecapKernel:
+                    if self.mfs_root_image:
+                        self.extra_kernels_with_mfs.append("CHERI_PURECAP_DE4_MFS_ROOT")
+                        self.extra_kernels_with_mfs.append("CHERI_PURECAP_DE4_MFS_ROOT_FUZZ")
+                        self.extra_kernels_with_mfs.append("CHERI_PURECAP_DE4_MFS_ROOT_BENCHMARK")
                 else:
-                    prefix = "BERI_DE4_"
+                    if self.crosscompile_target.is_hybrid_or_purecap_cheri():
+                        prefix = "CHERI128_DE4_"
```

Comment:
```suggestion
                        prefix = "CHERI_DE4_"
```

Reply:
Did we also rename all the FPGA configuration files?